hnn_core/externals/lfp.py

```diff
@@ -0,0 +1,206 @@
+"""
+LFPsim - Simulation scripts to compute Local Field Potentials (LFP) from cable
+compartmental models of neurons and networks implemented in NEURON simulation
+environment.
+
+LFPsim works reliably on biophysically detailed multi-compartmental neurons
+with ion channels in some or all compartments.
+
+Last updated 12-March-2016
+Developed by :
+Harilal Parasuram & Shyam Diwakar
+Computational Neuroscience & Neurophysiology Lab, School of Biotechnology,
+Amrita University, India.
+
+Email: harilalp@am.amrita.edu; shyam@amrita.edu
+www.amrita.edu/compneuro
+
+translated to Python and modified to use use_fast_imem by Sam Neymotin
+based on mhines code
+
+References
+----------
+1 . Parasuram H, Nair B, D'Angelo E, Hines M, Naldi G, Diwakar S (2016)
+Computational Modeling of Single Neuron Extracellular Electric Potentials
+and Network Local Field Potentials using LFPsim. Front Comput Neurosci 10:65
+[PubMed].
+"""
+
+from neuron import h
+
+import matplotlib.pyplot as plt
+
+
+def get_all_sections(sec_type='Pyr'):
+    ls = h.allsec()
+    ls = [s for s in ls if sec_type in s.name()]
+    return ls
+
+
+class LFPElectrode:
+    """LFP electrode class.
+
+    Parameters
+    ----------
+    coord : tuple
+        The (x, y, z) coordinates of the LFP electrode.
+    sigma : float
```

Comment:
I think folks might want access to this parameter. Any way we can expose it? Wouldn't hurt to expose "method" as well.